userspace/libsinsp/event.h

```diff
@@ -205,14 +205,6 @@ class SINSP_PUBLIC sinsp_evt : public gen_event
 		return m_info->flags;
 	}
 
-	/*!
-	\brief Get the event's category.
-	*/
-	inline ppm_event_category get_info_category() const
```

Comment:
I'm not sure about killing an API at this phase; we could use this to return higher bits perhaps?

Reply:
Uhm that's a good point, we can leave it and call under the hood `get_category`

Reply:
Done :) I've deprecated the API